src/components/Cewl/Cewl.tsx

```diff
@@ -162,121 +234,138 @@ const Cewl = () => {
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                ></InstallationModal>
             )}
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <Switch
-                    size="md"
-                    label="Advanced Mode"
-                    checked={checkedAdvanced}
-                    onChange={(e) => setCheckedAdvanced(e.currentTarget.checked)}
-                />
-                <TextInput
-                    label={"Target URL"}
-                    placeholder={"https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit"}
-                    required
-                    {...form.getInputProps("url")}
-                />
-                <TextInput
-                    label={"Save wordlist"}
-                    placeholder={"/home/kali/Desktop/Wordlist.txt"}
-                    {...form.getInputProps("wordlist")}
-                />
-                {checkedAdvanced && (
-                    <>
-                        <TextInput
-                            label={"Max depth"}
-                            type="number"
-                            placeholder={"Deafult is set to 2"}
-                            {...form.getInputProps("depth")}
-                        />
-                        <div style={{ display: "flex", alignItems: "center" }}>
+
+            <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
+                <LoadingOverlay visible={loading} />
+                {loading && (
+                    <div>
+                        <Button variant="outline" color="red" style={{ zIndex: 1001 }} onClick={handleCancel}>
+                            Cancel
+                        </Button>
+                    </div>
+                )}
+                <Stack>
+                    <Switch
+                        size="md"
+                        label="Advanced Mode"
+                        checked={checkedAdvanced}
+                        onChange={(e) => setCheckedAdvanced(e.currentTarget.checked)}
+                    />
+                    <TextInput
+                        label={"Target URL"}
+                        placeholder={"https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit"}
+                        required
+                        {...form.getInputProps("url")}
+                    />
+                    <TextInput
+                        label={"Save wordList"}
+                        placeholder={"/home/kali/Desktop/WordList.txt"}
+                        {...form.getInputProps("wordList")}
+                    />
+                    {checkedAdvanced && (
+                        <>
                             <TextInput
-                                label={"Min Length"}
-                                placeholder={"0"}
+                                label={"Max depth"}
```

Comment:
There is inconsistency with the use of capitalisation of the 2nd word. For example, should it be "Max Depth" or "Max depth"? For the next field you use capital "L" for "Min Length".

Reply:
done